dist/rpm/libxjwt.spec

```diff
@@ -1,5 +1,5 @@
 Name:           libxjwt
-Version:        1.0.3
+Version:        1.0.2
```

Comment:
i think this version should be 1.0.3, because this it he version in `master`.  when we tag it, and create the release tarball, then it will have the right version too?